components/sup/src/event.rs

```diff
@@ -93,15 +84,15 @@ pub struct EventStreamConfig {
     pub site:               Option<String>,
     pub meta:               EventStreamMetadata,
     pub token:              EventStreamToken,
-    pub url:                Address,
+    pub url:                SocketAddr,
```

Comment:
I think we need to move away from `SocketAddr` and just use a string.  `SocketAddr` will convert domain names to an IP address. If the nats service is answering on a domain name and has a server certificate signed against that CN which in almost all cases it will, then this will fail with a TLS connection error.